2-Browsers/Week1/homework/ex1-bookList/style.css

```
@@ -1 +1,17 @@
-/* Write your style here */
+
+li {
+  display: inline-block;
+  width: 450px;
+  margin: 15px;
+  text-align: center;
+  list-style: none;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
```

You can actually use the `margin` property here, which is a shorthand to set the margin on all four sides. The `margin` property goes from top, right, bottom, left (clockwise order). If three values are used: top, left and right, bottom. So in your code you only want to set left, right, and bottom so you can do this:

```suggestion
  margin: 0 auto 25px;
```

Info: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

---

Thanks for this suggestion